PC/hu_moments_gather/hu_moments_gather.cpp

```diff
@@ -20,26 +23,57 @@ int canny_threshold;
 const int threshold_ratio = 2;
 const int sobel_kernel_size = 3;
 
+int show_images_cli_arg = 0;
 
 int main(int argc, char** argv)
 {
-    if(argc < 2)
+    if(argc < 3)
     {
-        std::cout << "Usage: ./hu_moments_gather <input_image> <init_canny_threshold>\n";
+        std::cout << "Usage: ./hu_moments_gather <input_images_list_file> <init_canny_threshold> [show_images]\n";
         exit(1);
     }
 
-    if(argc == 3)
+    if(argc >= 3)
```

Comment:
Shouldn't be:
`if(argc == 3)` ???
Having 4 arguments we will not initialize
`show_images_cli_arg`, will we?